From 11d74fa5a22fe626c5e5a010b703cd46a136f253 Mon Sep 17 00:00:00 2001
From: Michael J Rubinsky <mrubinsk@horde.org>
Date: Mon, 14 Dec 2015 09:27:09 -0500
Subject: [PATCH] Escape form value.

Even though this is a numeric field, this isn't enforced until
the form is submitted.
---
 framework/Core/lib/Horde/Core/Ui/VarRenderer/Html.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/framework/Core/lib/Horde/Core/Ui/VarRenderer/Html.php b/framework/Core/lib/Horde/Core/Ui/VarRenderer/Html.php
index 62ae5599915..580dc27409d 100644
--- a/framework/Core/lib/Horde/Core/Ui/VarRenderer/Html.php
+++ b/framework/Core/lib/Horde/Core/Ui/VarRenderer/Html.php
@@ -48,7 +48,7 @@ protected function _renderVarInput_number($form, &$var, &$vars)
         return sprintf('<input type="text" size="5" name="%s" id="%s" value="%s"%s />',
                        htmlspecialchars($var->getVarName()),
                        $this->_genID($var->getVarName(), false),
-                       $value,
+                       htmlspecialchars($value),
                        $this->_getActionScripts($form, $var)
                );
     }
